### PR TITLE
Basic Info FadeIn State

### PR DESCRIPTION
**Pull request by Yashita101**

### CODE REVIEW

File: src/OnboardingSPA/pages/Steps/BasicInfo/stylesheet.scss
Comment (officiallygod) [inline src/OnboardingSPA/pages/Steps/BasicInfo/stylesheet.scss:6]: Can we get rid of all this CSS as we won't be needing it right?

Comment (Yashita101) [reply src/OnboardingSPA/pages/Steps/BasicInfo/stylesheet.scss:6]: Yes, removing it

File: src/OnboardingSPA/pages/Steps/BasicInfo/stylesheet.scss
Comment (officiallygod) [inline src/OnboardingSPA/pages/Steps/BasicInfo/stylesheet.scss:10]: Do we need this `__loadState`  CSS?

**Review by officiallygod — APPROVED** [review]

**Comment by officiallygod** [comment]
Let's wait for PRESS 2-462 to be merged before we merge this Story, as this has changes from that story.

**Comment by arunshenoy99** [comment]
Investigate if we can wait on the site logo to load as well instead of waiting only on flow and social API's

**Review by arunshenoy99 — CHANGES_REQUESTED** [review]
Pending: Investigate if we can wait on the site logo to load as well instead of waiting only on flow and social API's

File: src/OnboardingSPA/pages/Steps/BasicInfo/basicInfoForm.js
Comment (officiallygod) [inline src/OnboardingSPA/pages/Steps/BasicInfo/basicInfoForm.js:135]: Can we use Double Quotes as I saw in the docs usage.
Screenshot 2023-01-16 at 3 41 20 PM

**Review by officiallygod — COMMENTED** [review]
Also in basicInfoForm.js 
function setDefaultData() {
		if ( isLoaded ) {
			setSiteLogo( flowData?.data.siteLogo );
			setSiteTitle( flowData?.data.blogName );
			setSiteDesc( flowData?.data.blogDescription );
		}
	}
	
	Can we add a `?` after **data** too for all three cases? In case flow API fails??

**Comment by Yashita101** [comment]
> Pending: Investigate if we can wait on the site logo to load as well instead of waiting only on flow and social API's

The rendering of the image of the user depends on their internet speed, which is not possible to track via our code, as we can verify if the url is valid and not whether the rendering is complete. Hence this condition for siteLogo cannot be checked.

**Comment by arunshenoy99** [comment]
> > Pending: Investigate if we can wait on the site logo to load as well instead of waiting only on flow and social API's
> 
> The rendering of the image of the user depends on their internet speed, which is not possible to track via our code, as we can verify if the url is valid and not whether the rendering is complete. Hence this condition for siteLogo cannot be checked.

Have some idea around tracking, will create a new story.

**Review by arunshenoy99 — CHANGES_REQUESTED** [review]
@Yashita101 I just merged `trunk` into this branch to get a good idea of the changes and noticed something: can we please remove all the changes in `src/OnboardingSPA/pages/Steps/Ecommerce/StepAddress/index.js`? This change deletes `Animate` from `StepAddress` and we definitely need `Animate` in our address step.

**Review by arunshenoy99 — APPROVED** [review]
Looks good, will create a story with details to handle the site logo render.

**Pull request merged**
